asm-iap/Kptfile

```diff
@@ -50,6 +50,20 @@ openAPI:
         setter:
           name: anthos.servicemesh.trustDomainAliases
           listValues:
+    io.k8s.cli.setters.gcloud.container.cluster.ingress.httpPort:
+      type: integer
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.ingress.httpPort
+          value: "31224"
+          isSet: true
+    io.k8s.cli.setters.gcloud.container.cluster.ingress.statusPort:
+      type: integer
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.ingress.statusPort
+          value: "31223"
+          isSet: true
```

Comment:
This is like the nittiest nit ever, but can you be consistent with the order of these? In istio-operator the statusPort comes first.